packages/suite/src/views/settings/general/LabelingDisconnect.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { capitalizeFirstLetter } from '@trezor/utils';
+import { ActionColumn, SectionItem, TextColumn, ActionButton } from '@suite-components/Settings';
+import { useSelector, useActions } from '@suite-hooks';
+import * as metadataActions from '@suite-actions/metadataActions';
+import { Translation } from '@suite-components';
+import { useAnchor } from '@suite-hooks/useAnchor';
+import { SettingsAnchor } from '@suite-constants/anchors';
```

Comment:
I like how you separate with a line the import of `React` like a external library from the rest that are "our code"

Reply:
I think that it is more readable. 
Dan will probably add https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md to eslint so we will have it in all the files.